_posts/2024-11-11-test-markdown.md

```
@@ -0,0 +1,157 @@
+		Username:    "root", // 用户名
+		Password:    "root", // 密码
+	})
+	if err != nil {
+		return nil, err
+	}
+	return cli, nil
+}
+
+// 获取键值
+func getKeyValue(cli *clientv3.Client, key string) (string, int64, error) {
+	resp, err := cli.Get(context.Background(), key)
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)